responsibleai/responsibleai/_managers/counterfactual_manager.py

```diff
@@ -575,7 +580,23 @@ def _get_counterfactual_schema(version):
             return json.load(f)
 
     def list(self):
-        pass
+        """List information about the CounterfactualManager.
```

Comment:
wow, how has this list method not been implemented yet :sweat_smile: is there anything else we are still missing?

Reply:
Hopefully this is the last one.